ergo-lib/src/wallet.rs

```diff
@@ -41,6 +43,16 @@ impl From<TxSigningError> for WalletError {
 }
 
 impl Wallet {
+    /// Create wallet instance loading secret key from mnemonic
+    pub fn from_mnemonic(mnemonic_phrase: &str, mnemonic_pass: &str) -> Wallet {
```

Comment:
I think this is likely wrong. I have a feeling that we might need to implement `ExtendedPrivateKey` but would love to get clarification before going down that rabbit hole @greenhat 😄 

My other feeling is extended private keys might only be needed when we start generating addresses so maybe this is OK for just signing transactions etc

Reply:
No, this is fine for the start. It looks similar to https://github.com/ergoplatform/ergo/blob/95139ec990fc91f8f4c70a61468edf8834143329/src/main/scala/org/ergoplatform/nodeView/wallet/ErgoWalletService.scala#L283-L299 

And yes, we're going to need ext private keys when start generating(deriving) addresses.